src/DynamoRevit/ViewModel/RevitWatch3DViewModel.cs

```diff
@@ -210,20 +212,18 @@ private IEnumerable<GeometryObject> Tessellate(Curve curve)
             curve.Tessellate(pkg, renderPackageFactory.TessellationParameters);
 
             // get necessary info to enumerate and convert the lines
-            var lineCount = pkg.LineVertexCount * 3 - 3;
+            //var lineCount = pkg.LineVertexCount * 3 - 3;
             var verts = pkg.LineStripVertices.ToList();
 
             // we scale the tesselation rather than the curve
             var conv = UnitConverter.DynamoToHostFactor(UnitType.UT_Length);
 
-            // add the revit Lines to geometry collection
-            for (var i = 0; i < lineCount; i += 3)
+            var scaledXYZs = new List<XYZ>();
+            for (var i = 0; i < verts.Count; i+= 3)
             {
-                var xyz0 = new XYZ(verts[i] * conv, verts[i + 1] * conv, verts[i + 2] * conv);
-                var xyz1 = new XYZ(verts[i + 3] * conv, verts[i + 4] * conv, verts[i + 5] * conv);
-
-                result.Add(Line.CreateBound(xyz0, xyz1));
+                scaledXYZs.Add(new XYZ(verts[i] * conv, verts[i + 1] * conv, verts[i + 2] * conv));
             }
+            result.Add(PolyLine.Create(scaledXYZs));
```

Comment:
@Randy-Ma @mjkkirschner As it turns out, `Polyline.Create` is incredibly tolerant of small lines. It does not seem subject, at least when created lines for the background preview, to the short curve tolerance normally applied when drawing other types of curves in Revit. For this reason, I've switched this entire method of drawing curves, including straight curves, over to using that method.